app/javascript/notifications/types/Notifications__Entries.res

```diff
@@ -0,0 +1,11 @@
+type t =
+  | Unloaded
+  | PartiallyLoaded(array<Notifications__Entry.t>, string)
+  | FullyLoaded(array<Notifications__Entry.t>)
+
+let toArray = t =>
+  switch t {
+  | Unloaded => []
+  | PartiallyLoaded(entries, _) => entries
+  | FullyLoaded(entries) => entries
+  }
```

Comment:
We should probably create a functor for this - we've written this exact same thing in a bunch of places, and we're going to keep repeating this pattern. 🤔